kubernetes/cray-keycloak/Chart.yaml

```diff
@@ -3,4 +3,4 @@ description: Deploys Keycloak for Shasta
 keywords:
 - keycloak
 name: cray-keycloak
-version: 2.0.0
+version: 2.0.1
```

Comment:
should be a minor version update since this is added functionality: https://semver.org/

Reply:
That probably makes more sense as this is not a bug fix.  Updating to 2.1.0